packages/aws-otel-collector/aws-otel-collector.spec

```
@@ -0,0 +1,78 @@
+%global goproject github.com/aws-observability
+%global gorepo aws-otel-collector
+%global goimport %{goproject}/%{gorepo}
+
+Name: %{_cross_os}aws-otel-collector
+Version: 0.40.0
+Release: 1%{?dist}
+Summary: An AWS supported service to send telemetry data
+License: Apache-2.0
+URL: https://github.com/aws-observability/aws-otel-collector
+Source0: %{gorepo}-v%{version}.tar.gz
+Source1: aws-otel-collector.service
+Source2: aws-otel-collector-tmpfiles.conf
+Source3: aws-otel-collector-default-config.yaml
+
+# Change log file path from /opt to /var/log
+Patch0001: 0001-logger-change-collector-log-file-path.patch
+
+BuildRequires: %{_cross_os}glibc-devel
+Requires: %{name}(binary)
+
+%description
+%{summary}.
+
+%package bin
+Summary: Telemetry collector binary
+Provides: %{name}(binary)
+Requires: (%{_cross_os}image-feature(no-fips) and %{name})
+Conflicts: (%{_cross_os}image-feature(fips) or %{name}-fips-bin)
+
+%description bin
+%{summary}.
+
+%package fips-bin
+Summary: Telemetry collector binary, FIPS edition
+Provides: %{name}(binary)
+Requires: (%{_cross_os}image-feature(fips) and %{name})
+Conflicts: (%{_cross_os}image-feature(no-fips) or %{name}-bin)
+
+%description fips-bin
+%{summary}.
+
+%prep
+%autosetup -n %{gorepo}-%{version} -p0001
```

nit: this `-p0001` usage seems to be new - here, `socat`, `amazon-ssm-agent` - and I don't prefer it.

It invites the reader to think that it's related to the `Patch0001` patch, but it's actually an argument to the `patch` command that controls prefix stripping. It's weird to pad numeric arguments to CLI programs with leading zeroes.
```suggestion
%autosetup -n %{gorepo}-%{version} -p1
```

---

Ack, I'll change this here and also include commits to `socat` and `amazon-ssm-agent` replacing this.